<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AutoTTS</class>
 <widget class="QMainWindow" name="AutoTTS">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AutoTTS</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,1">
      <item>
       <widget class="QTextEdit" name="TextToRead">
        <property name="toolTip">
         <string>请以空格分隔词语</string>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,1,1000000000,1,1,1,1,1,1,1,1">
        <item>
         <widget class="QPushButton" name="ReadButton">
          <property name="text">
           <string>读出文字</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="StopButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>停止</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="RepeatLabel">
            <property name="toolTip">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;两次读词之间间隔 1 秒&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="text">
             <string>重复次数</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="RepeatInput">
            <property name="minimum">
             <number>1</number>
            </property>
            <property name="maximum">
             <number>5</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="VolumeLabel">
            <property name="text">
             <string>音量</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="SelectedVolumeLabel">
            <property name="text">
             <string>100</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="VolumeSlider">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="SeparatingTimeLabel">
            <property name="text">
             <string>间隔时间（秒）</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="SelectedTimeLabel">
            <property name="toolTipDuration">
             <number>-1</number>
            </property>
            <property name="layoutDirection">
             <enum>Qt::LeftToRight</enum>
            </property>
            <property name="text">
             <string>5</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSlider" name="SeparatingTimeSlider">
          <property name="minimum">
           <number>5</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="pageStep">
           <number>5</number>
          </property>
          <property name="value">
           <number>5</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="VoiceLabel">
          <property name="text">
           <string>选择语音</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="VoiceSelector"/>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>选择朗读引擎</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QRadioButton" name="EdgeTTSSelectionButton">
               <property name="text">
                <string>在线（Edge TTS）</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="LocalTTSSelectionButton">
               <property name="text">
                <string>本地（系统）</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>17</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="ImportButton"/>
    <addaction name="SaveButton"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="HelpButton"/>
    <addaction name="AboutButton"/>
    <addaction name="AboutQtButton"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="ImportButton">
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="SaveButton">
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="HelpButton">
   <property name="text">
    <string>帮助</string>
   </property>
  </action>
  <action name="AboutButton">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="AboutQtButton">
   <property name="text">
    <string>关于 Qt</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
